README.md

```diff
@@ -1,43 +1,80 @@
+4. Once the "dump.sql" file is created, from the root directly of the project in your terminal, run the command "docker-compose --build 	up pg". This will read the "pg" service in the "docker-compose.yaml" file and only build that image and run a container, whlie 			ignoring the other services. Once the image is build, the Postgres container will see the "dump.sql" file and begin copying the 		information to the database. This process should take very little time if you set the number of iterations to 0 or 1 earlier.
+	You will notice the creation of another folder called "pg" in the root directory of the project, which is just a volume to persist the data that was seeded so that you do not need to seed the database when stopping and restarting a Postgres container.
+
+### Option 1
+```console
+cd server/dumps
+chmod 777 catScript.sh
+npm run shell
+```
 
+### Option 2
+From the rootdir, paste the command that printed to your terminal. This command will vary depending on how many iterations you choose to run in "server/dataGeneration/generateDump.js". It should look like the following below:
+```console
+cat server/dumps/head.sql server/dumps/copySpaces.sql server/dumps/spacesBody0.sql server/dumps/spacesBody1.sql server/dumps/copyReviews.sql server/dumps/reviewsBody0.sql server/dumps/reviewsBody1.sql server/dumps/copyCalculations.sql server/dumps/calculationsBody0.sql server/dumps/calculationsBody1.sql server/dumps/foot.sql > server/dumps/dump.sql
+```
 
-## Table of Contents
+### Running the Postgres container
+From the rootdir, enter in your terminal
+```console
+docker-compose --build up pg
+```
 
-1. [Usage](#Usage)
-1. [Requirements](#requirements)
-1. [Development](#development)
+NOTE: The initial copying of the dum.sql may take a little time, but all subsequent containers will be instant. To stop and delete the container, enter:
+```console
+docker-compose down
+```
 
-## Usage
+NOTE: Postgres has the "out-of-box" capability of generating a dump.sql file to back up existing databases. The entire strategy of this data generation script was to create a file using the same syntax as a native pg_dump file, as these scripts follow several recommendations from the Postgres documentation on speeding up the process of seeding large datasets. More can be read [here](https://www.postgresql.org/docs/9.1/populate.html)
 
-> Some usage instructions
+##PgAdmin
+In the docker-compose.yml file, you will see that I have also included a service to run a container for PgAdmin (the Postgres GUI). To see it in action, run the following command:
+```console
+docker-compose up --build pg pgadmin
+```
 
-## Requirements
+In the browser, go to http://localhost:8080, and login with 
 
-An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
+email: user@mail.com
+password: password
 
-- Node 6.13.0
-- etc
+1. Create a server
 
-## Development
+![create-server](https://sdc-perlman.s3.amazonaws.com/pgadmin/createServer.png)
```

Comment:
Is this a command to be run? Maybe use the code formatting

Reply:
This the markdown language syntax to display and image in the README, but here in the code review it shows it how I typed it